### ISSUE TITLE
[24] Selected filter is not available to screen readers

### ISSUE THREAD

**Issue by gordcorp** [AusDTO/dta-website#297]
This issue was identified as part of the accessibility testing performed on the DTA website Jan 2017: https://drive.google.com/open?id=0B3SXZ9dr83v0Y2cwV21HdUZ1STQ

Applies: Filters on Assessment reports
Type: WCAG failure – 4.1.2 Name, Role Value
Platform: Desktop and mobile
Severity: Medium

Sighted users can visually identify the selected filter but this information is not conveyed to screen reader users.
We note that the heading below the filters (e.g. All assessment reports) does identify which filter is active. However, providing information to screen reader users about which filter is selected while they are navigating through the options provides a much better experience.

![image](https://cloud.githubusercontent.com/assets/2324569/23935830/7b7d7de0-09a2-11e7-8fc5-e07b3232fc8a.png)

**Recommendation**

Update the visually hidden text from ‘Show’ to ‘Showing’ for the filter which is selected.

For example, if All reports is selected, update the visually hidden text to ‘Showing’ so a screen reader users will hear ‘Link. Showing All reports.’. When another filter is selected, change the visually hidden text back to ‘Show’ for this item, and update to ‘Showing’ for the selected filter.

**Sample code**
```
<li>
 <a href="#all" class="is-current"><span
 class="is-visuallyhidden">Showing </span>All reports</a>
</li>
```

**Comment by bjgordon** [comment]
Do their suggestion. Plus add aria tags:
- aria-selected for the selected item
- role=tablist for ul
- role=tab for each li 
refer to http://accessibility.athena-ict.com/aria/examples/tabpanel2.shtml
